src/storage/mvcc/reader/util.rs

```diff
@@ -11,12 +11,8 @@ pub enum CheckLockResult {
     /// Key is locked. The key lock error is included.
     Locked(Error),
 
-    /// Key is not locked.
+    /// Key is not locked or the lock is ignored.
     NotLocked,
-
-    /// Key's lock exists but was ignored because of requesting the latest committed version
-    /// for the primary key. The committed version is included.
-    Ignored(u64),
```

Comment:
Now `CheckLockResult` can be just alias to `Result<Error>` I think.

Reply:
Yes. I will change it.